### DISCUSSION TITLE
What drivers Oracle Airbyte connectors uses?

### DISCUSSION THREAD

**Discussion by MichalisDBA** [airbytehq/airbyte#31768] [Connector Questions] [answered]
Hello.

What drivers for Oracle Databases (Source / Destination) does AirByte use?

Is it JDBC? Native linux InstantClient? NPM Oracle driver package? Some other package?

Where are these files located?

Thank you.

**Comment by marcosmarxm** [comment, marked as answer]
It uses JDBC driver
https://github.com/airbytehq/airbyte/blob/228a0712d7b7e5eabaf607c3a086d500bbe70cc9/airbyte-integrations/connectors/source-oracle/build.gradle#L26

**Reply by MichalisDBA** [reply]
Thank you for your answer. Is it possible to build a new Oracle connector based on the existing one but only change the JDBC version of the driver to use an old one for legacy oracle databases? Is there a walkthrough on how to do it?

**Reply by marcosmarxm** [reply]
There isn't any documentation for that. You can read Airbyte connector code and give a try.
